spec/aruba/api_spec.rb

```diff
@@ -698,7 +698,7 @@
       context 'when file_name contains fixtures "%" string' do
         let(:runtime) { instance_double('Aruba::Runtime') }
         let(:config) { double('Aruba::Config') }
-        let(:environment) { instance_double('Aruba::Environment') }
+        let(:environment) { object_double(Aruba::Environment.new) }
```

Comment:
You're absolutely right, @mattwynne .  Except in this situation, ```instance_double``` would no longer work on ```Aruba::Environment``` because it's now a ```SimpleDelegator``` and as such, Rspec won't verify the ```Aruba::Environment``` double. (Using ```instance_double```, RSpec will report that 'environment (Aruba::Environment.new) can't respond to *clear*' and other methods that are delegated to the ```*Environment``` classes.  )  
If there is a way that RSpec can verify it, I can turn it back in to an ```instance_double```  
Should there be a short comment at the end of the new line 701 explaining this? Something like ```# object_double required because Aruba::Environment is a SimpleDelegator``` ?